components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_uint64_to_unicode_string(path: &str, json: &str) -> String;
```

Comment:
Looks like there is some inconsistent indentation here - could we run `cargo fmt` over these files?

Reply:
done